x/gamm/pool-models/stableswap/pool.go

```diff
@@ -258,11 +258,21 @@ func (p *Pool) CalcJoinPoolShares(ctx sdk.Context, tokensIn sdk.Coins, swapFee s
 	return pCopy.joinPoolSharesInternal(ctx, tokensIn, swapFee)
 }
 
+// TODO: implement this
```

Comment:
Do we have an issue for this?

Reply:
https://github.com/osmosis-labs/osmosis/issues/2376